src/main/javacc/ASTID.java

```diff
@@ -7,12 +7,20 @@ public ASTID(int id) {
     super(id);
   }
 
-  public void setName(String n) {
-    name = n;
+  public void setName(String name) {
+    this.name = name;
   }
 
   public String getName() {
-    return name;
+    return this.name;
+  }
+
+  public String getFormattedName() {
```

Comment:
can you do the lowercasing in the copy.jjt file instead? It looks like the parser there distinguishes between an identifier and a quoted identifier. So in line 259/265 where the identifier() function is implemented, you could do the lowercasing there

Reply:
Done.